src/runtime-rs/config/configuration-qemu-coco-dev-runtime-rs.toml.in

```diff
@@ -356,62 +355,63 @@ pflashes = []
 #
 # If set to the empty string "", no extra monitor socket is added. This is
 # the default.
-#extra_monitor_socket = "hmp"
+extra_monitor_socket = ""
 
 # Disable the customizations done in the runtime when it detects
 # that it is running on top a VMM. This will result in the runtime
 # behaving as it would when running on bare metal.
 #
-#disable_nesting_checks = true
+# Default false
+disable_nesting_checks = false
 
 # This is the msize used for 9p shares. It is the number of bytes
 # used for 9p packet payload.
-#msize_9p = @DEFMSIZE9P@
+msize_9p = @DEFMSIZE9P@
```

Comment:
Just to check this value. When I did a check of the code I found:
```
KATA_ANNO_CFG_HYPERVISOR_MSIZE_9P => match self.get_value::<u32>(key) {
                        Ok(v) => {
                            hv.shared_fs.msize_9p = v.unwrap_or_default();
                        }
                        Err(_e) => {
                            return Err(u32_err);
                        }
                    },
```
and
```
 if self.msize_9p == 0 {
                    self.msize_9p = default::DEFAULT_SHARED_9PFS_SIZE_MB;
                }
```
and
```
pub const DEFAULT_SHARED_9PFS_SIZE_MB: u32 = 128 * 1024;
```
so was that the default before?